config.yaml

```diff
@@ -2,12 +2,7 @@ baseurl: "https://engineering.rakuten.today/"  # Include trailing slash
 languageCode: "en-us"
 title: "Rakuten Engineering Blog"
 pygmentsstyle: 'monokai'
-
-# Theme settings
-# theme: novela
-module:
-  imports:
-    - path: "github.com/forestryio/hugo-theme-novela"
+theme: github.com/forestryio/hugo-theme-novela
```

Comment:
how's the `module.imports` way of pulling the theme different from the `theme` way?
Either way work for me, just curious. 
@boazy I think you set that up originally, right? or did we inherit that from the forestry template?

Reply:
If you use `theme` directly, doesn't it mean the theme won't be pulled by itself? I'm a little rusty on Hugo modules.

Reply:
Look like the way above is the standard way:
https://gohugo.io/hugo-modules/use-modules/#use-a-module-for-a-theme

Reply:
@jcayzac's way probably works, but it is undocumented.
It seems the current code is also incorrect though: the `# theme: novella` part is commented out, so it would probably use the default theme.

Reply:
So, at first I misunderstood and thought `theme` would just work with no need for `go.mod`, but I was wrong. It doesn't work just by itself, it still uses the module imported by `go.mod`. It looks to me that `theme` really is just a shorthand for adding an entry to `module.imports` and that both things are thus equivalent.

Reply:
@boazy 

> @jcayzac's way probably works, but it is undocumented.

It's actually [how hugo-theme-novela tells you to install the theme in its README](https://github.com/forestryio/hugo-theme-novela#install-from-the-command-line)/